### PR TITLE
Fix #12 - Check testing query parameter before cookies.

### PR DESCRIPTION
**Pull request by rlr**
r?

### CODE REVIEW

**Comment by jsocol** [comment]
Does this let you reset the testing status (e.g., set it to `1` and then change it to `0` with another request with the GET param)? If so, can you add a test for that?

File: waffle/tests/test_waffle.py [hunk 1]
```diff
@@ -243,7 +243,9 @@ def test_testing_flag(self):
         assert 'foo' in request.waffle_tests
         assert request.waffle_tests['foo']
 
+        # GET param should override cookie
         request = get(dwft_foo='0')
+        request.COOKIES['dwft_foo'] = True
```
Comment (jsocol) [inline waffle/tests/test_waffle.py]: The cookie should be the string `'True'`, not the actual boolean.

**Comment by rlr** [comment]
Yeah, that is why I added the cookie to the existing test making sure the GET parameter was overriding it. Do you want something in addition to that?

File: waffle/tests/test_waffle.py (hunk 1 above)
Comment (rlr) [inline waffle/tests/test_waffle.py]: Ah, yes!

**Comment by jsocol** [comment]
Just a test to demonstrate that you can use the query string to set the testing cookie to one value, then use the query string to set it to another value (doesn't work on master, you have to clear the cookie--I think this patch makes that work).

**Comment by rlr** [comment]
Not sure if new notifications notifies you of updates to the branch now. Anyway, does that test make sense ^

**Comment by jsocol** [comment]
Yay! Whole new (fixed) functionality with the same 2-line code change, thanks! Squashing and merging.

**Comment by jsocol** [comment]
78c34c3789dfdae0272c300da702324659800646

**Pull request closed without merging**